packages/shared-ui/src/styles/index.css

```diff
@@ -1,3 +1,4 @@
+@import url('https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@300..700&display=swap');
```

Comment:
I have not noticed any differences for Inter, if there are then we should do this for Inter as well. 
For now, 300 to 700 is fine since our normal weight is 400 and bold weight is 700. The difference in texts for web-js and react application was due to us not fetching this font.